common/src/main/java/com/wynntils/core/functions/FunctionManager.java

```diff
@@ -30,6 +33,13 @@ private static void registerFunction(Function<?> function) {
         if (function instanceof ActiveFunction<?> activeFunction) {
             activeFunction.init();
         }
+        // FIXME: This is sort of hacky. We should have these as ActiveFunctions instead,
```

Comment:
We already have enabled functions stored though?

Reply:
So, I cheated a bit here, partially because I was not sure how to continue with the dependency change I proposed.

There is really no point in having a separate DependentFunction the way I did it, I realized half-way through. We need to enable them, to register the manager dependeny. Without enabling, the manager is not necessarily loaded. 

Problem is, I don't know what we really say about non-loaded managers. What happen if I call ActionBarModel.getMana(), even if we have not registered ActionBarModel as a dependency? Will it fail? Will we just get random bad data back? The idea we had for ActiveFunctions with "well, try return what you got and if it's bad, it's bad", is not really working if we try calling a manager/model, and it is allowed to crash since it's not properly enabled.

Reply:
So for the time being, I register the manager dependencies when loading the functions. That is no good if we really want to be able to not load all managers. But I think we need more clarity on what to expect if you're interacting with a manager that is not loaded. If that is okay, guaranteed to not crash, but can give dummy data, then this is fine.

But if they might crash, then we still will need to check if we're enabled, and if not, return null or "N/A" or whatever.

Reply:
So manager changes again?

Reply:
Yeah... question is what to change. Maybe it's just a declaration, stating that managers must work without crashing, and at most deliver random data, when not enabled.